x-pack/plugin/security/qa/multi-cluster/src/javaRestTest/java/org/elasticsearch/xpack/remotecluster/RemoteClusterSecurityEsqlIT.java

```diff
@@ -324,20 +330,23 @@ public void testCrossClusterQuery() throws Exception {
         populateData();
 
         // query remote cluster only
-        Response response = performRequestWithRemoteSearchUser(esqlRequest("""
+        Request request = esqlRequest("""
             FROM my_remote_cluster:employees
             | SORT emp_id ASC
             | LIMIT 2
-            | KEEP emp_id, department"""));
-        assertOK(response);
+            | KEEP emp_id, department""");
+        Response response = performRequestWithRemoteSearchUser(request);
+        assertRemoteOnlyResults(response);
+
+        // same as above but authenticate with API key
+        response = performRequestWithRemoteSearchUserViaAPIKey(request);
```

Comment:
this isn't related to the change here ... i just forgot to test this via API key in the prior PR.